recipe/meta.yaml

```diff
@@ -67,25 +64,26 @@ requirements:
     - zlib
     # requirements specified by the package itself, see
     # github.com/tensorflow/tensorflow/blob/v{{ version }}/tensorflow/tools/pip_package/setup.py
-    - absl-py >=0.10.0
-    - astunparse >=1.6.3
-    - python-flatbuffers 1.12.*
-    - google-pasta >=0.2
-    - h5py >=2.10
-    - keras-preprocessing >=1.1.2
+    - absl-py >=0.10,<1
+    - astunparse >=1.6.3,<1.7
+    - python-flatbuffers >=1.12,<2
+    - google-pasta >=0.2,<1
+    - h5py >=3.1.0,<3.2
+    - keras-preprocessing >=1.1.2,<1.2
     - numpy 1.19.*
-    - opt_einsum 3.3.0.*
+    - opt_einsum >=3.3,<3.4
     - protobuf >=3.9.2
-    - six 1.15.0.*
-    - termcolor >=1.1.0
-    - typing_extensions 3.7.4.*
-    - wheel >=0.35
-    - wrapt >=1.11.1
+    - six >=1.15,<1.16
+    - termcolor >=1.1,<1.2
+    - typing_extensions >=3.7.4,<3.8
+    - wheel >=0.35,<1
+    - wrapt >=1.12.1,<1.13
     # upstream wants exact pin for gast
-    - gast ==0.3.3
+    - gast ==0.4.0
     # TF-API needs to move in sync
-    - tensorboard >=2.4.0,<2.5
-    - tensorflow-estimator >=2.4.0,<2.5
+    - tensorboard >=2.6,<3
+    # - tensorflow-estimator >=2.6,<3
+    - keras >=2.6,<3
```

Comment:
Should this be `<2.7`?

Reply:
No, as far as I understood the `~=` operator this is correct: https://github.com/tensorflow/tensorflow/blob/919f693420e35d00c8d0a42100837ae3718f7927/tensorflow/tools/pip_package/setup.py#L106